services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -1148,6 +1150,15 @@ protected void initP2PNetwork() {
         new LocalOperationAcceptedFilter<>(p2pNetwork::publishSignedBlsToExecutionChange));
   }
 
+  protected void initEphemeryValidationService() {
+    Optional<Eth2Network> network = beaconConfig.eth2NetworkConfig().getEth2Network();
+    if (network.isPresent() && network.equals(Optional.of(Eth2Network.EPHEMERY))) {
+      final EphemerySlotValidationService slotValidationService =
+          new EphemerySlotValidationService();
+      eventChannels.subscribe(SlotEventsChannel.class, slotValidationService);
```

Comment:
one option would be defer this logic (basically the whole function) to start, so that the constructor creates a final EphemerySlotValidationService, and start only subscribes if the network is appropriate.
Then we could test for all other networks that start doesnt subscribe...
Stop can then also unsubscribe...